next/components/ui/RegulationListItem/RegulationListItem.tsx

```diff
@@ -65,6 +65,7 @@ export const RegulationListItem = ({
         </div>
 
         {mainDocumentHref && (
+          /** Cant find where to test this */
```

Comment:
This is "VZN" row card, specifically the button on the right.

You can find them here: https://bratislava.sk/mesto-bratislava/sprava-mesta/legislativa-mesta/vseobecne-zavazne-nariadenia

![image](https://github.com/bratislava/bratislava.sk/assets/19418224/a645d6fa-8347-4db7-96c1-7851994a9a1e)

Whole VZNs will undergo refactor, so do not use big effort here please :D

Reply:
Replaced for MLink and fixed TS error.